core-java/src/main/java/com/baeldung/networking/proxies/CommandLineProxyDemo.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.networking.proxies;
+
+import java.net.URL;
+import java.net.URLConnection;
+
+
+public class CommandLineProxyDemo {
+
```

Comment:
file seems to be using tabs to indent

Reply:
I've fixed the indentation and whitespace.  Thanks!  I forgot to bring in my baeldung settings when I switched to STS4.